src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pytest
+
+from diffpy.srmise.dataclusters import DataClusters
+
+
+@pytest.mark.parametrize(
```

Comment:
since we are only doing one test here, we don't need the mark.parameterize, you can just load the values directly into the test below.